### DISCUSSION TITLE
OpenBB SDK Playground [Beginner friendly - no local install needed]

### DISCUSSION THREAD

**Discussion by dMLTquant** [OpenBB-finance/OpenBB#4007] [Show and tell]
Hello, World! 👋

I hope this will be helpful to someone out there: [GitHub > dMLTquant > openbb_sdk_exporation](https://github.com/dMLTquant/openbb_sdk_exporation)

- It is a straightforward way of playing with and testing OpenBB SDK without installing anything on your local machine 🤩
- Requirements: GitHub account (free will do) and GitPod account (free will do)
- Just click on the `Open in GitPod` button from the README file and everything will run in your browser 😉

Hope you enjoy it and it helps out somehow

**Comment by jose-donato** [comment]
This is amazing. Good job 🙌

**Comment by DidierRLopes** [comment]
This is sick 🔥
